pkgs/development/hare-third-party/hare-json/default.nix

```
@@ -13,18 +13,10 @@ stdenv.mkDerivation (finalAttrs: {
 
   nativeBuildInputs = [ hare ];
 
-  configurePhase = ''
-    runHook preConfigure
-
-    export HARECACHE="$NIX_BUILD_TOP/.harecache"
-    export BINOUT="$NIX_BUILD_TOP/.bin"
-
-    makeFlagsArray+=(
-      PREFIX="${builtins.placeholder "out"}"
-    )
-
-    runHook postConfigure
-  '';
+  makeFlags = [
+    "HARECACHE=.cache"
```

Maybe keep it `.harecache`

```suggestion
    "HARECACHE=.harecache"
```

---

Done it.